sdk/core/Azure.Core.TestFramework/src/Azure.Core.TestFramework.csproj

```diff
@@ -10,6 +10,7 @@
     <PackageReference Include="NUnit" />
     <PackageReference Include="NUnit3TestAdapter" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
+    <PackageReference Include="Azure.ResourceManager" />  
```

Comment:
@m-nash do you have any concerns with the test framework depending on Azure.ResourceManager?

Reply:
I removed this.